### PR TITLE
Show orders data card in analytics hub

### PR DESCRIPTION
**Pull request by rfermontero**
Closes: #5237 (Part of)

### Description
This pull request uses previously created AnalyticsInformationCard to show specific order stats data into new analytics hub

### Testing instructions

1. Open the application compiled in debug mode
2. Click on analytics section on bottom navigation bar
3. Orders card should appear

### Images/gif

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/analytics/AnalyticsRepository.kt [hunk 1]
```diff
@@ -53,46 +55,67 @@ class AnalyticsRepository @Inject constructor(
...
-        when (dateRange) {
-            is DateRange.SimpleDateRange ->
-                fetchRevenueStats(dateRange.from.formatToYYYYmmDD(), dateRange.from.formatToYYYYmmDD(), granularity)
-            is DateRange.MultipleDateRange ->
-                fetchRevenueStats(
-                    dateRange.from.from.formatToYYYYmmDD(), dateRange.from.to.formatToYYYYmmDD(),
-                    granularity
-                )
+                    val previousOrdersCount = previousPeriodRevenue.getOrNull()!!.parseTotal()?.ordersCount ?: 0
+                    val previousOrderValue = previousPeriodRevenue.getOrNull()!!.parseTotal()?.avgOrderValue ?: 0.0
```
Comment (anitaa1990) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/analytics/AnalyticsRepository.kt:76]: I keep getting the error that `avgOrderValue` cannot be resolved. Is this a change in FluxC that needs to be updated in Woo?

Comment (rfermontero) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/analytics/AnalyticsRepository.kt:76]: Yes, sorry I forgot to comment it here.
I would need to update it but before we have to merge the next items:

https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2214
https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2215
https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2211

Comment (anitaa1990) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/analytics/AnalyticsRepository.kt:76]: I have reviewed PRs wordpress-mobile/WordPress-FluxC-Android#2214 and wordpress-mobile/WordPress-FluxC-Android#2215 👍 You can now update this PR with the commit hash from [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2211/commits/dab84321c7792c0bd64f9ed10289b261023749f1) and that should fix the build issue I'm facing

Comment (rfermontero) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/analytics/AnalyticsRepository.kt:76]: Thanks for the help!. I've updated the fluxc version with the last of the PR :)

**Review by anitaa1990 — COMMENTED** [review]
Nice job @colymore 🥇 I faced a couple of minor issues when testing.

- The currency is displayed in the total order section (I believe this is the count and does not require the currency?)
- The `See Report` button for the orders card is not clickable. Is that taken care of in another PR?
- There are some merge conflicts in the PR that need to be addressed 😄 
- There are a bunch of lint errors in the code.

**Comment by rfermontero** [comment]
> Nice job @colymore 🥇 I faced a couple of minor issues when testing.
> 
> * The currency is displayed in the total order section (I believe this is the count and does not require the currency?)
> * The `See Report` button for the orders card is not clickable. Is that taken care of in another PR?
> * There are some merge conflicts in the PR that need to be addressed 😄
> * There are a bunch of lint errors in the code.

Hi @anitaa1990  Thanks!
1. Totally right, I will remove it.
2. Yes, this would come from a merge with analytics-hub. Checking it
3 4 Fixing. :)

**Pull request merged**